packages/devtools_app/test/support/utils.dart

```diff
@@ -121,3 +124,60 @@ extension RichTextChecking on CommonFinders {
         (widget) => widget is RichText && widget.text.toPlainText() == text);
   }
 }
+
+/// Workaround to initialize the live widget binding with assets.
+///
+/// The [LiveTestWidgetsFlutterBinding] is useful for unit tests that need to
+/// perform true async operations such as communicating with the VM Service.
+/// Unfortunately the default implementation doesn't work with the patterns we
+/// use to load assets in the devtools application.
+/// TODO(jacobr): consider writing proper integration tests instead rather than
+/// using this code path.
+void initializeLiveTestWidgetsFlutterBindingWithAssets() {
```

Comment:
This is the key to running tests we would normally run using a DartVM using flutter test without running into trouble.